sktime/forecasting/statsforecast.py

```diff
@@ -14,58 +14,104 @@
 _check_soft_dependencies("statsforecast", severity="warning")
 
 
-class AutoARIMA(_StatsForecastAdapter):
+class StatsForecastAutoARIMA(_StatsForecastAdapter):
```

Comment:
why do we actually inherit from a `_StatsForecastAdapter`? Will there be more forecasting models that can inherit from it or only `AutoARIMA`?